src/aioquic/quic/congestion/base.py

```diff
@@ -17,15 +17,16 @@ class QuicCongestionControl(abc.ABC):
     congestion_window: int = 0
     ssthresh: Optional[int] = None
 
-    def __init__(self, *, max_datagram_size: int) -> None:
+    def __init__(self, *, max_datagram_size: int, recovery) -> None:
         self.congestion_window = K_INITIAL_WINDOW * max_datagram_size
+        self.recovery = recovery
```

Comment:
No, let's not have a reference loop please

Reply:
How should I get the smoothed RTT in another way ?

Reply:
Changed to latest RTT instead